terraform/modules/forest_node/user-data.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
```

Comment:
```suggestion
# This bash script is templated using <engine>, with the variables defined in <tfvars>.
# THIS MEANS THAT ${VARIABLES} YOU SEE ARE FOR THE TEMPLATE ENGINE, NOT BASH
#
# It is inserted to the newly created VM at <some path> as `user-data` by <orchestration>, and run <at startup>.
# TODO(samuelarogbonlo): refactor so we don't template the script itself.
```

I'm guessing at a lot of stuff here from the filename, and previous discussion, but this is the kind of information that should be contained in the code (or the README if it's really basic).

You've probably had to learn a lot about `terraform` and orchestration to write this code - why not save that knowledge for future you, or the next developer or reviewer ;)
That way, our code stays easier to maintain, review etc.

I'm also still not convinced by templating a bash script like this - scripts take _arguments_, they're not _templated_